client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -145,6 +145,7 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
       locations: util.List[PartitionLocation]): Unit = {
     val map = latestPartitionLocation.computeIfAbsent(shuffleId, newMapFunc)
     locations.asScala.foreach(location => map.put(location.getId, location))
+    invalidateLatestMaxLocsCache(shuffleId)
```

Comment:
Since cache invalidate when location updated,  so we can consider extending rpcCacheExpireTime to a longer duration, such as `60s`